packages/ckeditor5-coremedia-studio-integration/src/content/BlobDisplayServiceDescriptor.ts

```diff
@@ -1,15 +1,11 @@
-import { ServiceDescriptor } from "@coremedia/service-agent";
+import { serviceDescFactory, ServiceDescriptorWithProps } from "@coremedia/service-agent";
 import BlobDisplayService from "./BlobDisplayService";
 
 /**
  * Descriptor used to fetch service from service agent.
  */
-class BlobDisplayServiceDescriptor implements ServiceDescriptor<BlobDisplayService> {
-  constructor() {
-    this.name = "blobDisplayService";
-  }
-
-  name: string;
+export function createBlobDisplayServiceDescriptor(): ServiceDescriptorWithProps<BlobDisplayService> {
+  return serviceDescFactory<BlobDisplayService>({
+    name: "blobDisplayService",
+  });
```

Comment:
I think we decided to prefer const and arrow functions:

```suggestion
export const createBlobDisplayServiceDescriptor = (): ServiceDescriptorWithProps<BlobDisplayService> => {
  return serviceDescFactory<BlobDisplayService>({
    name: "blobDisplayService",
  });
```

(may require adapting to ESlint rules)

The same applies to other descriptor factory methods.